rpcs3/Emu/System.h

```diff
@@ -401,6 +401,25 @@ class Emulator final
 	static void SaveSettings(const std::string& settings, const std::string& title_id);
 };
 
+struct hle_locks_t
```

Comment:
For the love of god, stop dumping everything into System

Reply:
Fixed,

Reply:
thanks